src/server/components/formRunner/helpers.ts

```diff
@@ -132,8 +144,8 @@ export async function parseJsonFormData(
 ): Promise<Array<Partial<FormRunner.Question>>> {
   const formsJsonFile =
     isUnderTest === true
-      ? `/forms-json/${listType}.json`
-      : `../src/server/components/formRunner/forms-json/${listType}.json`;
+      ? `/forms-json/${kebabCase(listType)}.json`
```

Comment:
hmm.. do we need to think of a better solution to this? ideally you would have the json files in the correct format, because that's easier to change (rather than changing code). 

a lot of the query parameters use camelCase also, so maybe worth sticking it in another refactor/ticket?